docs/manuals/Installatiehandleiding/Installatiehandleiding.md

```diff
@@ -0,0 +1,5 @@
+# Installatiehandleiding
```

Comment:
So I actually think we should write this in English and not Dutch. This is because the target audience is not any member of a council (in the context of Dimpact anyway) as far as I know.

Reply:
Ok. Fixing this